Community_moderators.md

```diff
@@ -0,0 +1,472 @@
+# Community Moderators - The Good Docs Project

```

Comment:
minor suggestion: this file could be named like the others in this directory (either all caps, `COMMUNITY_MODERATORS.md`, or Pascal case, `CommunityModerators.md`)

Reply:
I made the change on my local copy, but for some reason it's not fully changing it in my commit. I'll see if I can fix this!